actions/translation-cursor-deploy/main.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Custom Cursor Deploy GitHub action
+ * @see {@link https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action}
+ *
+ * Updates the deployment translation cursor file in the S3 bucket.
+ */
+
+import * as core from '@actions/core'
+import {exec} from '@actions/exec'
+
+import {writeLineToFile, runAction, fileExistsInS3} from '../utils'
+
+runAction(async () => {
+    const bucket = core.getInput('bucket_name', {required: true})
+    const hash = core.getInput('hash', {required: true})
+
+    await cursorDeploy({
+        bucket,
+        hash
+    })
+})
+
+const PREVIOUS_HASH_COMMAND = 'previous' // special command for rollback to hash from /previous
+
+interface CursorDeployActionArgs {
+    bucket: string
+    hash: string
+}
+
+export async function cursorDeploy({bucket, hash}: CursorDeployActionArgs) {
+    const previousPath = `s3://${bucket}/translation-deploy/previous`
```

Comment:
Should we extract `translation-deploy/previous` and `translation-deploy/latest`  to constants?